### PR TITLE
Update readme files

### PR DESCRIPTION
**Pull request by HiddenBug**

### CODE REVIEW

File: pilz_robot_tfc_api/README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# PILZ robot test facility control API 
 
-
-
+This packages contains the API's for Python and C++,
```
Comment (martiniil) [inline pilz_robot_tfc_api/README.md]:
```suggestion
This packages contains the API's for Python and C++
```

**Review by martiniil — DISMISSED** [review]
Please commit my suggestion and merge.

**Comment by jschleicher** [comment]
Is the name clash of the repo with [pilz_testutils inside pilz_robots](https://github.com/PilzDE/pilz_robots/tree/melodic-devel/pilz_testutils) intended? Should this package(s) replace the former one?

**Comment by martiniil** [comment]
> Is the name clash of the repo with [pilz_testutils inside pilz_robots](https://github.com/PilzDE/pilz_robots/tree/melodic-devel/pilz_testutils) intended? Should this package(s) replace the former one?

Actually, it was not intended. But yes, the package `pilz_testutils` in `pilz_robots` can be dissolved in the future.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# pilz_testutils
+Test utilities used across different PILZ ROS packages.
+
+## Package `pilz_robot_tfc_api`
+[pilz_robot_tfc_api](pilz_robot_tfc_api/README.md) contains the API to
```
Comment (martiniil) [inline README.md]:
```suggestion
[pilz_robot_tfc_api](pilz_robot_tfc_api/README.md) contains the Python and C++ API's for
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# pilz_testutils
+Test utilities used across different PILZ ROS packages.
+
+## Package `pilz_robot_tfc_api`
+[pilz_robot_tfc_api](pilz_robot_tfc_api/README.md) contains the API to 
+programmatically control the robot test facility via test.
```
Comment (martiniil) [inline README.md]:
```suggestion
programmatically controlling the PILZ robot test facility via test.
```

File: pilz_robot_tfc_api/README.md [hunk 4]
```diff
@@ -1,3 +1,4 @@
+# PILZ robot test facility control API 
 
-
-
+This packages contains the API's for Python and C++
```
Comment (martiniil) [inline pilz_robot_tfc_api/README.md]:
```suggestion
This packages contains the Python and C++ API's for
```

File: pilz_robot_tfc_api/README.md [hunk 5]
```diff
@@ -1,3 +1,4 @@
...
+to programmatically control the PILZ robot test facility via test.
```
Comment (martiniil) [inline pilz_robot_tfc_api/README.md]:
```suggestion
programmatically controlling the PILZ robot test facility via test.
```

**Review by martiniil — CHANGES_REQUESTED** [review]

**Review by martiniil — APPROVED** [review]

**Pull request merged**